dokku-vm/azuredeploy.json

```diff
@@ -69,17 +100,16 @@
         },
         "allowedValues": [
             "East US",
-            "East US 2",
-            "East Asia",
-            "West US",
```

Comment:
What API are you calling. Here's what I see in this PR 
```
[
            "East US",
            "South Central US",
            "Central US",
            "North Europe",
            "West Europe",
            "Southeast Asia",
            "Japan West",
            "Japan East",
            "Australia Southeast",
            "Australia East"
       ]
```

Try calling this API - https://msdn.microsoft.com/en-us/library/azure/dn790540.aspx
`https://management.azure.com/subscriptions/<subscription-id>/locations?api-version={api-version}`